app/shared/components/design-system/discard-changes-modal/DiscardChangesModal.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import { Box, Button, Modal, Typography } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+
+import { styles } from './DiscardChangesModal.styles';
+
+interface DiscardChangesModalProps {
+  open: boolean;
+  handleClose: () => void;
+  handleSubmit: () => void;
+}
+
+const DiscardsChangesModal: React.FC<DiscardChangesModalProps> = ({ open, handleClose, handleSubmit }) => {
```

Comment:
```suggestion
const DiscardChangesModal: React.FC<DiscardChangesModalProps> = ({ open, handleClose, handleSubmit }) => {
```
Because you write it in filename without 's'

Reply:
And write tests for this component, please